docs/static/listing-a-plugin.asciidoc

```diff
@@ -6,10 +6,11 @@ If your plugin meets the <<plugin-acceptance,quality and acceptance guidelines>>
 
 The plugin source and readme will continue to live in your repo, and we will direct users there. 
 
-If you would like to have your plugin included in the {logstash-ref}[Logstash Reference]: 
-
-* verify that it meets our <<plugin-acceptance,quality and acceptance guidelines>>
-* create a new https://github.com/elasticsearch/logstash/issues[issue] in the Logstash repository. 
-** Use `PluginListing: <yourpluginname>` as the title for the issue.
-** Apply the `docs` label.  
-** In the body of the issue, explain the purpose and value your plugin offers, and describe how this plugin adheres to the guidelines.
+If you would like to have your plugin included in the {logstash-ref}[Logstash Reference], create a new https://github.com/elasticsearch/logstash/issues[issue] in the Logstash repository with the following information:
+* Title: `PluginListing: <your-plugin-name>`
+* Body:
+** Brief description of the plugin (what it is and what it does).
+** Link to the source code.
+** Link the to the README.md, docs/index.asciidoc or both.
```

Comment:
```suggestion
** Link to the README.md or docs/index.asciidoc.
```

Reply:
I prefer that we have one documentation link pointing to either readme or asciidoc file.  They can choose the one that they feel best represents their plugin.

Reply:
sounds good to me, btw... I copied that from the current docs :) :P